resources/authgear/templates/en/web/authflowv2/__html_head.html

```diff
@@ -10,6 +10,7 @@
 <meta name="x-cldr-locale" content="{{ $.ResolvedCLDRLocale }}">
 <meta name="x-color-scheme" content="{{ $.ColorScheme }}">
 <meta name="x-sentry-dsn" content="{{ $.AuthUISentryDSN }}">
+<meta name="x-window-message-allowed-origins" content="{{ $.AuthUIWindowMessageAllowedOrigins }}">
```

Comment:
Is it a common practice to put auth ui config in meta tag?

Reply:
I think this is the current way to do it.